docs/serverless-v2/06-01-function-cr.md

```diff
@@ -42,6 +41,16 @@ spec:
         return 'Hello ' + name;
       }
     }
+  status:
+    phase: Running
+    conditions:
+      [...]
+      - type: Deployed
+        lasttransitiontime: "2020-03-30T11:57:59+02:00"
+        reason: DeploySucceeded
+        message: ""
+    observedgeneration: 3
+    imagetag: dd5f487a-fc44-4f06-90d5-196dd7246b92
```

Comment:
The params names should be camel-cased, @m00g3n please verify if the implementation is also camel-cased...

Reply:
This is the sample taken from lates revision:
```
apiVersion: serverless.kyma-project.io/v1alpha1
kind: Function
metadata:
  annotations:
    kubectl.kubernetes.io/last-applied-configuration: |
      {"apiVersion":"serverless.kyma-project.io/v1alpha1","kind":"Function","metadata":{"annotations":{},"name":"demo225","namespace":"testme"},"spec":{"function":"module.exports = {\n    main: function(event, context) {\n      return 'Hello World'\n    }\n  }\n","functionContentType":"plaintext","runtime":"nodejs8","size":"S"}}
  creationTimestamp: "2020-04-08T06:19:10Z"
  generation: 1
  name: demo225
  namespace: testme
  resourceVersion: "891670"
  selfLink: /apis/serverless.kyma-project.io/v1alpha1/namespaces/testme/functions/demo225
  uid: 8bb281b6-4c88-4453-83c5-943b875d1c1a
spec:
  function: |
    module.exports = {
        main: function(event, context) {
          return 'Hello World'
        }
      }
  functionContentType: plaintext
  runtime: nodejs8
  size: S
status:
  conditions:
  - lastTransitionTime: "2020-04-08T06:19:11Z"
    reason: CreateConfigSucceeded
    type: Initialized
  - lastTransitionTime: "2020-04-08T06:20:21Z"
    reason: BuildSucceeded
    type: ImageCreated
  - lastTransitionTime: "2020-04-08T06:20:21Z"
    reason: CreateServiceSucceeded
    type: Deploying
  - lastTransitionTime: "2020-04-08T06:20:43Z"
    reason: DeploySucceeded
    type: Deployed
  imageTag: 59fb0423-4f4e-45e3-9553-04221bd7dc89
  observedGeneration: 1
  phase: Running
```